dotcom-rendering/src/components/Timeline.tsx

```diff
@@ -128,6 +128,15 @@ const eventStyles = css`
 	margin-bottom: ${space[5]}px;
 	position: relative;
 
+	${from.tablet} {
+		margin-left: -21px;
+		margin-right: -21px;
```

Comment:
The tablet dimension looks a lot narrower than the other elements…

Reply:
This narrow view on tablet is what we have currently for standard article types